src/dlite-storage.c

```diff
@@ -251,6 +251,7 @@ char **dlite_storage_uuids(const DLiteStorage *s, const char *pattern)
     char buf[DLITE_UUID_LENGTH+1];
     void *ptr, *iter = s->api->iterCreate(s, pattern);
     int n=0, len=0;
+    if (!iter) return NULL;
```

Comment:
I would move this line one up, no reason to declare n and len if you return

Reply:
Er ikke det noe som kompileroptimeringen fikser automatisk? 

Selv om det ikke er nødvendig i C99 (som dlite er basert på), liker jeg personlig å deklarere variabler i toppen av funksjonen så at deres type er lett å finne. Er det en dårlig praksis i moderne kode?

Reply:
Agreed, did not notice this was the declare section. Maybe a newline before the if?